bentoml/_internal/bento/docker/entrypoint.sh

```diff
@@ -18,6 +18,7 @@ _main() {
 	# Overide the BENTOML_PORT if PORT env var is present. Used for Heroku
 	if [[ -v PORT ]]; then
 		echo "\$PORT is set! Overiding \$BENTOML_PORT with \$PORT ($PORT)"
+		echo "NOTE: Yatai also set \$PORT, hence \$BENTOML_PORT will be overridden."
```

Comment:
move this to the inline comments above ? No need to log it

Reply:
done